### DISCUSSION TITLE
XPs and roles

### DISCUSSION THREAD

**Discussion by 0xsingular** [pkgxdev/pkgx#888] [Help]
As a question that came up on the discord server, for some reason XPs and roles are not assigned for engagement and contributions (content, social engagement, GitHub), at the moment only discord posts are considered, XPs for which are automatically assigned.

It would be a solution to create a public table where all contributions+links and their corresponding XPs will be included.
That way anyone could check at any time if their contributions have been considered and XPs will be assigned.
I offer to moderate and add all contributions and XPs to this table.

**Comment by devranjan** [comment]
That could be a great solution for the current problem where XP points are not getting assigned for various kinds of engagements. We need a tool for track those activities.

**Reply by rieyuone** [reply]
nice

**Comment by Dinesh200114** [comment]
That will be nice , but I seems all are working good now .

**Comment by Claire3653** [comment]
Amazing!
In this way we can increase community engagement!

**Comment by rajatj86** [comment]
what project is coming  on Tea Protocol

**Comment by dorikunz** [comment]
I think this is the best way.

**Comment by uvice** [comment]
This Tea protocol looks promising. I just got onboard today
